dynamiqs/utils/wigners.py

```diff
@@ -15,6 +15,8 @@
 __all__ = ['wigner']
 
 
+@ft.partial(jnp.vectorize, signature='(n,m)->(k),(l),(k,l)', excluded={1, 2, 3})
+@ft.partial(jax.jit, static_argnums=(3, 4))
```

Comment:
Should we reverse both? I don't know if vectorize jits by default.

Reply:
It's sensitively equivalent